pom.xml

```diff
@@ -247,7 +239,7 @@
                     <artifactId>maven-javadoc-plugin</artifactId>
                     <version>3.1.0</version>
                     <configuration>
-                        <failOnError>false</failOnError>
+                        <failOnError>true</failOnError>
```

Comment:
Have you tried to generate javadoc to ensure no javadoc failures and this won't break build?

Reply:
I finally found out where you addressed this comment via your email explanation. Thanks